src/services/assetService.ts

```diff
@@ -130,8 +136,8 @@ export class AssetService {
     /**
      * Get assets from provider
      */
-    public async getAssets(): Promise<IAsset[]> {
-        return await this.assetProvider.getAssets();
+    public async getAssets(projectName: string): Promise<IAsset[]> {
```

Comment:
The assetProvider already has a reference to the project so we shouldn't need to pass in the project name as a function argument.